python/semantic_kernel/connectors/ai/open_ai/exceptions/content_filter_ai_exception.py

```diff
@@ -0,0 +1,116 @@
+# Copyright (c) Microsoft. All rights reserved.
+
+from dataclasses import dataclass
+from enum import Enum
+from typing import Any, Dict
+
+from openai import BadRequestError
+
+from semantic_kernel.connectors.ai.ai_exception import AIException
+
+
+class ContentFilterResultSeverity(Enum):
+    High = "high"
```

Comment:
Python considers these constants, so the names should be all caps: `HIGH = "high"`

Reply:
source: https://docs.python.org/3/howto/enum.html

Reply:
Fixed